### ISSUE TITLE
How to interpret the results from pareidolia?

### ISSUE THREAD

**Issue by Linhua-Sun** [cmdoret/pareidolia#6]
Hi, now I can run pareidolia well. But how should I interpret the results from `pareidolia`? What does `diff_score` mean? 
As you previously mentioned `a signal-to-noise ratio (replicate variance < condition variance) and a percentile threshold` in https://github.com/koszullab/chromosight/issues/47. Can pareidolia output more details to aid the filter process? Sorry for the question since there is now no detailed documentation. I am planing to use pareidolia and chromosight for a loop analysis manuscript.
```
chrom1  start1   end1     chrom2  start2   end2     bin1  bin2  diff_score
1       20000    30000    1       50000    60000    2     5     0.16421711095037572
1       120000   130000   1       260000   270000   12    26    0.2865824780793555
1       310000   320000   1       380000   390000   31    38    0.15063850456637157
1       700000   710000   1       730000   740000   70    73    0.07630547845442173
1       940000   950000   1       980000   990000   94    98    0.15368042829125256
1       1270000  1280000  1       1290000  1300000  127   129   0.27051386683066125
1       1370000  1380000  1       1790000  1800000  137   179   0.0716760862212018
1       1360000  1370000  1       2810000  2820000  136   281   0.12005183124843921
1       1510000  1520000  1       1550000  1560000  151   155   0.12930835156337844
```
Thanks!

**Comment by cmdoret** [comment]
Hi @Linhua-Sun 

Thanks for the feedback, I have included a written and graphical description of the algorithm in the readme, along with what each threshold does. In the latest version (v0.6.0), the output will contain an additional column with the signal-to-noise-ration (snr) to inform about the reliability of the call. This is the ratio between inter-condition change and intra-condition variability.

I have removed the percentile threshold, instead I use a density threshold to filter out low coverage regions (explained in the readme).
